src/libexpr/eval-gc.cc

```diff
@@ -110,18 +111,18 @@ void fixupBoehmStackPointer(void ** sp_ptr, void * _pthread_id)
 #    else
 #      error "Need one of `pthread_attr_get_np` or `pthread_getattr_np`"
 #    endif
-    if (pthread_attr_getstack(&pattr, &osStackLow, &osStackSize)) {
+    if (pthread_attr_getstack(&pattr, &osStackLimit, &osStackSize)) {
         throw Error("fixupBoehmStackPointer: pthread_attr_getstack failed");
     }
     if (pthread_attr_destroy(&pattr)) {
         throw Error("fixupBoehmStackPointer: pthread_attr_destroy failed");
     }
 #  endif
-    osStackBase = (char *) osStackLow + osStackSize;
+    osStackBase = (char *) osStackLimit + osStackSize;
     // NOTE: We assume the stack grows down, as it does on all architectures we support.
     //       Architectures that grow the stack up are rare.
-    if (sp >= osStackBase || sp < osStackLow) { // lo is outside the os stack
-        sp = osStackLow;
+    if (sp >= osStackBase || sp < osStackLimit) { // lo is outside the os stack
```

Comment:
nit:

Does this `lo` refer to `sp`, `osStackBase`, or `osStackLimit` (nee `osStackLow`)? (Might be worth updating the comment while you're here)

Reply:
Good point. It's `sp`.
`lo` might have made sense in the bdwgc code, maybe, but not here.
Thanks

Reply:
Thank you!